examples/payout_usage.php

```diff
@@ -0,0 +1,108 @@
+<?
+
+use BTCPayServer\Client\PullPayment;
+
+class PullPayments {
+
```

Comment:
the three common variables I would add as class variables if you use a class like this (instead of repeating them in each function), in the other examples we do not use class but just procedural code, but works either way
```
  public $storeId = '';
  public $host = '';
  public $apiKey = '';
```

Reply:
`require __DIR__ . '/../vendor/autoload.php';

use BTCPayServer\Client\PullPayment;

class PullPayments {

    public $apiKey;
    public $host;
    public $storeId;

    public function __construct()
    {   
        $this->apiKey = '';
        $this->host = '';
        $this->storeId = '';
    }`

Like this?